pylint_odoo/checkers/no_modules.py

```diff
@@ -371,11 +377,21 @@ def colon_list_to_dict(self, colon_list):
         """
         return dict([item.split(":") for item in colon_list])
 
+    def replace_and_first_lower(self, string):
+        """ Conver 'Hello World' to 'hello_World'
+
+            :param string: String to convert
+            :returns: string
+            :rtype: string
+        """
+        return (string[:1].lower() + string[1:]).replace(' ', '_')
```

Comment:
Check built-in string methods for python for this case:
e.g. str.capitalize or str.title